### PR TITLE
EN-4323 hotfix

### PR DESCRIPTION
**Pull request by aerust**

### CODE REVIEW

File: store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala [hunk 1]
```diff
@@ -133,23 +125,69 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
...
+    logger.info("droping copy (datasetInfo: {}, secondaryCopyInfo: {}, cookie: {})",
+      datasetInfo, secondaryCopyInfo, cookie)
+    withPgu(dsInfo, Some(datasetInfo)) { pgu =>
+      doDropCopy(pgu, datasetInfo, secondaryCopyInfo, isLatestCopy)
+      pgu.commit()
+    }
     cookie
   }
 
+  def doDropCopy(pgu: PGSecondaryUniverse[SoQLType, SoQLValue], datasetInfo: DatasetInfo,
```
Comment (chitang) [inline store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala]: Move doDropCopy to a handler class like the other events.

File: store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala [hunk 2]
```diff
@@ -133,23 +125,69 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
...
-
   // Is only ever called as part of a resync.
-  def dropCopy(datasetInternalName: String, copyNumber: Long, cookie: Secondary.Cookie): Secondary.Cookie = {
-    // TODO: dropCopy
-    logger.warn(s"TODO: dropCopy '${datasetInternalName}' (cookie: ${cookie}})")
+  def dropCopy(datasetInfo: DatasetInfo,
+               secondaryCopyInfo: SecondaryCopyInfo,
+               cookie: Secondary.Cookie, isLatestCopy: Boolean): Secondary.Cookie = {
+    logger.info("droping copy (datasetInfo: {}, secondaryCopyInfo: {}, cookie: {})",
+      datasetInfo, secondaryCopyInfo, cookie)
```
Comment (chitang) [inline store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala]: indent to where the arguments start in the previous line.

File: store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala (hunk 1 above)
Comment (aerust) [inline store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala]: But, it's not really an event. It's like resync. and the doResync is in this file. Keeping it here seems more consistent to me?

**Comment by chitang** [comment]
lgtm

**Pull request merged**